### ISSUE TITLE
Allow me to move between music services

### ISSUE THREAD

**Issue by steren** [dtinit/data-transfer-project#495]
Allow me to move:
- my tracks
- my playlists
- my music preferences and tastes

between:
- Spotify
- Youtube Music
- Google Play Music
- Deezer
- ... (and other on-demand music services)

**Comment by nealot** [comment]
I think "listen history" is too vague to be generically used by different engines. Wouldn't just "history" suffice?

**Comment by danbri** [comment]
Some combination of https://schema.org/ListenAction and https://www.w3.org/TR/activitystreams-core/ would probably cover much of the listening aspect. For the "what was listened to", in Schema.org we have a design for music metadata based heavily on the MusicBrainz project and the Music Ontology, and lean on sites like MusicBrainz or Wikidata.org for entity identifiers (for MusicGroup, https://schema.org/MusicAlbum https://schema.org/MusicRecording etc.). The example from @csuwildcat sketches how this might look as a JSON(-LD) payload, using https://schema.org/MusicGroup ...

Presumably if you wanted to transfer out derrived data structures (like trained machine learning models) there's a way to ship those around too.

**Comment by Poopooracoocoo** [comment]
Yeah I started using YTM and the lack of last.fm support for instance sucks. Ideally every service would be more open...

**Comment by lisad** [comment]
We do have playlist transfer functionality as part of DTP now, so closing this done.
